.travis.yml

```diff
@@ -15,15 +15,6 @@ _base_envs:
 jobs:
   fast_finish: true
   include:
-    - env:
-      - PYTHON=2.7
-      - NUMPY=1.14.1
-      - PANDAS=0.22.0
-      - *test_and_lint
-      - *no_coverage
-      - *no_optimize
-      - *no_imports
```

Comment:
Does anyone mind me dropping this CI entry as we add one for 3.7?  cc @jakirkham @jcrist

Reply:
I do. We generally try to support both Python 2/3 in our codebase as there are inevitably scientific libraries that we want to pull in that haven't made the switch. Though I think this has gotten better in the past year or two, it's still a surprisingly common problem (even with new codes).

Reply:
There is still another 2.7 entry.  We used to have two.  You might want to check the full file in this branch

Reply:
@jakirkham does your concern still hold, given that there is another Python 2.7 entry?

Reply:
Oops. Sorry. Missed the other one. This is fine.